### PR TITLE
Template merge

### PR DESCRIPTION
**Pull request by mattumland**
## Is this a fix or a feature?
Feature
## What is the change?
This adds the Wall and Artwork components and connects them to the data being fetched in App. It renders then with some initial salon styling but it isn't finalized. I will create a new card for future improvement to the salon styling. 
## Where should the reviewer start?
Review the new files (Wall.js/scss Artwork.js/scss). Neither are very long. 
## How should this be tested?
Run the dev server, watch how it loads. Try reloading a few times to see how it different collections of art appear.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "cypress": "cypress open"
+    "cypress": "cypress open",
+    "sass": "sass src/Sass:src/Css --watch --no-source-map"
```
Comment (percworld) [inline package.json:21]: Would enjoy hearing more about this and wonder if I need to ```npm run sass```

File: src/components/App/App.js [hunk 2]
```diff
@@ -44,18 +45,24 @@ function App() {
...
     <div className="App">Salon
-    
-      {ids.length && console.log('Rendering IDs: ', ids)}
-      {wall.length && console.log('WALL: ', wall)}
-      {wall.length && <img src={wall[0].primaryImageSmall} />}
-      {wall[1] && <img src={wall[1].primaryImageSmall} />}
-      {wall[2] && <img src={wall[2].primaryImageSmall} />}
-      {wall[3] && <img src={wall[3].primaryImageSmall} />}
-      {wall[4] && <img src={wall[4].primaryImageSmall} />}
+
```
Comment (percworld) [inline src/components/App/App.js:56]: Great! We'll have the ability to make this dynamic when state lives in wall

File: src/index.css [hunk 3]
```diff
@@ -1,13 +1,3 @@
-body {
```
Comment (percworld) [inline src/index.css:1]: Did we install a normalize file yet? I can't remember, but wanted to mention it

**Review by percworld — COMMENTED** [review]
I'm curious if we still need a normalize file. Did anyone do this yet? Might be a good time if not. Thanks for the wonderful grid!

File: package.json (hunk 1 above)
Comment (mattumland) [reply package.json:21]: Technically, that command will ensure that changes to the sass file will live update. I find that most of the time it does it on its own. But if it's not working at any time, you can run that command in a separate terminal and it should ensure that sass updates auto refresh.

File: src/index.css (hunk 3 above)
Comment (mattumland) [reply src/index.css:1]: I installed normalize ✅

File: package.json (hunk 1 above)
Comment (percworld) [reply package.json:21]: Thanks for clearing this up!

File: src/index.css (hunk 3 above)
Comment (percworld) [reply src/index.css:1]: Great I appreciate all of this.

**Comment by rdtho2525** [comment]
+ Landing page renders random art pieces (as expected) after multiple reloads
+ No apparent request errors in the console
+ As discussed, each image renders twice

**Pull request merged**